src/components/navigation/navigation.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './navigation.css';
+import Home from '@/icons/home';
+import Bookmark from '@/icons/bookmark';
+import Compass from '@/icons/compass';
+import NavElement from '@/UI/nav/nav';
+const navLinks = [
+  {
+    content: <Home width={'24'} height={'24'} />,
+    key: 'Home',
+    path: `/home/currentUser`,
+  },
+  {
+    content: <Compass width={'24'} height={'24'} />,
+    key: 'Explore',
+    path: '/explore',
+  },
+  {
+    content: <Bookmark width={'24'} height={'24'} />,
```

Comment:
Why not name this variable 'icon' instead of 'content'? It is closer to sense in my opinion

Reply:
Because not only icons but also text will be flipped inside this block